### DISCUSSION TITLE
Elitemobs Compatibility

### DISCUSSION THREAD

**Discussion by M3RGeo** [QuickShop-Community/QuickShop-Hikari#655] [Ideas]
Would it be possible to add compatibility with Elitemobs.
I have both plugins on my server, but we can't make the shops available for everyone to use at the moment as with elitemobs you get a soulbound enchantment that's applied to all loot it generates, which stops players from picking up the loot of another player. at the moment this specific enchantment makes it so players can be scammed with the shops. as you can buy an elite item and it will just lie on the ground. i tried making an exception with them in the config lore-blacklist but the plugin still accepts any elite items, had a few players try to sell the items and they all succeeded in getting a shop made with different items(all soulbound to them). we can bypass this, but then the player loot can be stolen by other players.

**Comment by Ghost-chu** [comment]
Here we go:

Snapshot build for QuickShop-Hikari: https://ci.codemc.io/job/Ghost-chu/job/QuickShop-Hikari-SNAPSHOT/1153/artifact/quickshop-bukkit/target/QuickShop-Hikari-3.6.0.2.jar

Snapshot build for EliteMobs compatibility module: https://ci.codemc.io/job/Ghost-chu/job/QuickShop-Hikari-SNAPSHOT/1153/artifact/compatibility/elitemobs/target/Compat-EliteMobs-3.6.0.2.jar

You need to install the snapshots both of them.

NOTE: I had NEVER test them because I didn't use EliteMobs, but it should be technically working.

**Comment by M3RGeo** [comment]
Thanks, I just saw the new release on spigot, on my way to install them both. github didn't send me a notification so didn't think you saw the message yet.

**Comment by M3RGeo** [comment]
Thanks, I updated and tested the new version and compatibility module, and they work nicely. will now be able to grant players access to chestshops.
